share/functions/__fish_cancel_commandline.fish

```diff
@@ -3,16 +3,10 @@ function __fish_cancel_commandline
     set -l cmd (commandline)
     if test -n "$cmd"
         commandline -C 1000000
-        # TODO: Switch from using tput and standout mode to `set_color` when themes have been
-        # augmented to include support for background colors or has support for standout/reverse
-        # mode.
-        #
-        # Set reverse fg/bg color mode, output ^C, restore normal mode, clear to EOL (to erase any
-        # autosuggestion).
+        echo -ns (set_color $fish_color_cancel) "^C" (set_color normal)
         if command -sq tput
-            echo -ns (set_color -r) "^C" (set_color normal) (tput el; or tput ce)
-        else
-            echo -n "^C"
+            # Clear to EOL (to erase any autosuggestion).
```

Comment:
Is it possible you misconstrued the diff? This feature was present before too; check out the comment at old line 10.

I just extracted the `^C` echoing out of the conditional statement since it's independent of `tput` availability. It was apparently written that way initially because `tput` was used instead of `set_color`, but it was changed in 1fb92b6.